e2e/tests/functional/plugins/telemetryTable/telemetryTable.e2e.spec.js

```diff
@@ -45,6 +45,86 @@ test.describe('Telemetry Table', () => {
     await expect(rows).toHaveCount(50);
   });
 
+  test('on load, auto scrolls to top for descending, and to bottom for ascending', async ({
+    page
+  }) => {
+    async function getScrollPosition(top = true) {
+      const tableBody = page.locator('.c-table__body-w');
+
+      // Wait for the scrollbar to appear
+      await tableBody.evaluate((node) => {
+        return new Promise((resolve) => {
+          function checkScroll() {
+            if (node.scrollHeight > node.clientHeight) {
+              resolve();
+            } else {
+              setTimeout(checkScroll, 100);
+            }
+          }
+          checkScroll();
+        });
+      });
```

Comment:
can we move this outside of the test body? That would also remove the need for the eslint ignore on conditionals in tests